web-src/src/components/App.js

```diff
@@ -24,7 +24,7 @@ import { MainSidePanel } from './MainSidePanel.js';
 import { PromptSessionPanel } from './PromptSessionPanel.js';
 import { PromptTemplateLibraryPanel } from './PromptTemplateLibraryPanel.js';
 import { viewTypeState, ViewType } from '../state/ViewType.js';
-import { mainSidePanelState, MainSidePanelType } from '../state/MainSidePanelState.js';
+import { mainSidePanelTypeState, MainSidePanelType } from '../state/MainSidePanelTypeState.js';
 import { FavoriteVariantListPanel } from './FavoriteVariantListPanel.js';
 import { useShellContext } from './ShellProvider.js';
 
```

Comment:
Please import newly introduced localization files and update `function getAllMessages(locale)`.

Reply:
I'd say this is an unrelated change, as this import wasn't introduced in this PR.

Reply:
I mean:

**`import * as messagesContentFragmentExportButton from './__localization__/ContentFragmentExportButton.l10n.js';`**

and

```
function getAllMessages(locale) {
  if (typeof locale !== 'string') {
    return {};
  }

  const normalizedLocale = locale.replace(/-/g, '_');
  return {
    ...messagesApp[normalizedLocale],
    ...messagesMainSidePanel[normalizedLocale],
    ...messagesPromptResultCard[normalizedLocale],
    ...messagesPromptSessionSideView[normalizedLocale],
    ...messagesFavorites[normalizedLocale],
    ...messagesImageViewer[normalizedLocale],
    ...messagesContentFragmentExportButton[normalizedLocale],
  };
}
```

Reply:
Thank you, @askayastha22 , for helping me navigate the complexities of l10n :-)